compiler/luci/import/include/luci/Import/GraphBuilder.h

```diff
@@ -33,6 +33,12 @@ class GraphBuilder : public GraphBuilderBase
 public:
   virtual ~GraphBuilder() = default;
 
+  // common validate to check number of inputs and single output
+  bool validate(const ValidateArgs &args, size_t i) const
```

Comment:
How about `validate_input_cnt` or `validate_common_inout` or `is_single_output_with` or something else?
`validate` seems generous.

Reply:
Yes, it seems general... 
Behind this is that I like short names and `size_t i` argument will distinguish existing method...

Reply:
I see it :)
Then `bool validate(const ValidateArgs &args, size_t input_cnt) const` seems okay?
I like short names too but if both function name and parameter name iare too short, sometimes it is confused :(
If you don't like this too, let's go as it is :)

Reply:
`input_cnt` seems OK :)